tasks/human_eval_077.rs

```diff
@@ -5,11 +5,139 @@ HumanEval/77
 /*
 ### VERUS BEGIN
 */
+use vstd::arithmetic::mul::*;
+use vstd::math::abs;
 use vstd::prelude::*;
 
 verus! {
 
-// TODO: Put your solution (the specification, implementation, and proof) to the task here
+fn is_cube(x: i32) -> (ret: bool)
+    requires
+        x != i32::MIN,  // avoid overflow: -(i32::MIN) == (i32::MAX) + 1
+
+    ensures
+        ret <==> exists|i: int| 0 <= i && abs(x as int) == #[trigger] (i * i * i),
```

Comment:
Is it possible to show that this is equivalent to this?
```
ret <==> exists|i: int| x == #[trigger] (i * i * i),
```
That's what I was initially expecting to see, given the English prompt.